README.md

```diff
@@ -109,6 +123,24 @@ Hovering over any resource, class declaration or other Puppet symbol provides in
 
 ![hover_support](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/hover_support.gif)
 
+### Outline View
+
+The new `Outline View` shows the symbol tree of the currently selected Puppet manifest. This currently supports classes, types, variables and parameters. More symbols are planned to be added.
+
+![outline_view](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/outline_view.gif)
+
+### Breadcrumbs
+
+When the `breadcrumbs.enabled` setting is set to true, both the file path and the symbol tree hierarchy are shown in the Breadcrumb view.
+
+![breadcrumbs](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/breadcrumbs.gif)
+
+### Go to Symbol
+
+Opening the `Command palatte` and typing the `@` symbol initiates the `Go to symbol` view which allows you to navigate around inside a Puppet manifest quickly.
```

Comment:
Palette and symbol should both be capitalized for consistency.

Suggest "... quickly" to "... more quickly"